example/presto_cluster/main.tf

```diff
@@ -81,81 +109,84 @@ module "postgres" {
   source = "github.com/fredrikhgrelland/terraform-nomad-postgres.git?ref=0.3.0"
 
   # nomad
-  nomad_datacenters = local.nomad_datacenters
-  nomad_namespace   = local.nomad_namespace
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
   nomad_host_volume = "persistence-postgres"
 
   # postgres
   service_name    = "postgres"
   container_image = "postgres:12-alpine"
   container_port  = 5432
-  vault_secret = {
-    use_vault_provider     = false,
-    vault_kv_policy_name   = "",
-    vault_kv_path          = "",
-    vault_kv_username_name = "",
-    vault_kv_password_name = ""
-  }
-  admin_user                      = "hive"
-  admin_password                  = "hive"
+
+  # Vault provided credentials
+  vault_secret = local.postgres_vault_secret
+  # Credentials will be provided via vault > vault_secret.use_vault_provider = true
+  admin_user                      = "postgres"
+  admin_password                  = "postgres"
+
   database                        = "metastore"
-  container_environment_variables = ["PGDATA=/var/lib/postgresql/data"]
   volume_destination              = "/var/lib/postgresql/data"
   use_host_volume                 = true
   use_canary                      = true
+  container_environment_variables = ["PGDATA=/var/lib/postgresql/data/"]
 }
 
 module "hive" {
   source = "github.com/fredrikhgrelland/terraform-nomad-hive.git?ref=0.3.0"
 
+  depends_on = [
+    module.minio,
+    module.postgres
+  ]
+
   # nomad
-  nomad_datacenters  = local.nomad_datacenters
-  nomad_namespace    = local.nomad_namespace
+  nomad_datacenters  = ["dc1"]
+  nomad_namespace    = "default"
   local_docker_image = false
 
   # hive
-  use_canary          = true
-  hive_service_name   = "hive-metastore"
-  hive_container_port = 9083
-  hive_docker_image   = "fredrikhgrelland/hive:3.1.0"
+  use_canary                           = true
+  hive_service_name                    = "hive-metastore"
+  hive_container_port                  = 9083
+  hive_docker_image                    = "fredrikhgrelland/hive:3.1.0"
+  hive_container_environment_variables = ["SOME_EXAMPLE=example-value"]
+
   resource = {
-    cpu    = 500,
+    cpu    = 500
     memory = 1024
   }
-  resource_proxy = {
-    cpu    = 200,
-    memory = 128
-  }
 
-  #support CSV -> https://towardsdatascience.com/load-and-query-csv-file-in-s3-with-presto-b0d50bc773c9
-  #metastore.storage.schema.reader.impl=org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader
-  hive_container_environment_variables = [
-    "HIVE_SITE_CONF_metastore_storage_schema_reader_impl=org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader"
-  ]
+  resource_proxy =  {
+    cpu     = 200
+    memory  = 128
+  }
```

Comment:
```suggestion
```
Defaults, so they can be removed.

Reply:
I would like to set up all possible variables `explicitly` in the example, even though they are default.